### ISSUE TITLE
viewSpec - cannot exit when annotating

### ISSUE THREAD

**Issue by ChelseaD9** [jonkatz2/monitoR#7]
I am using the viewSpec function to annotate a 1-hour recording using the following code and then  typing "a" to enter annotation mode.  Everything works until I try to exit.

viewSpec(clip = "X:Kaleidoscope/FN1663_2018-03-08_053700_EST.wav",interactive = T,start.time = 1,units = "minutes",page.length = 30,annotate = T,output.dir = "X:Kaleidoscope",player = wmp)

It says to "right click twice to exit" but when I right click twice I invoke an RStudio drop down menu that gives me the option to Reload or Inspect.

Is there another way to exit annotation mode?  

Thanks!

Chelsea

**Comment by jonkatz2** [comment]
I'm not an RStudio user, but I remember there is a different method within their graphics viewer. Is there a stop sign button at the top of the pane you can click, or some other interactive button?

**Comment by ChelseaD9** [comment]
Yes!  On the upper right corner of the plot window a small white button that says finish pops up.  When I clicked on that it exited from annotation mode.

Thank you so much for the speedy response!
